pkg/distributors/test/MultiRewards.test.ts

```diff
@@ -88,6 +88,14 @@ describe('Staking contract', () => {
     });
   });
 
+  it('reverts if a rewarder attempts to setRewardsDuration before adding a reward', async () => {
```

Comment:
Can we also add a test for the newly added allowlist modifier?

Reply:
Done - test failure appears to be a flicker on the pools package (unrelated to this test case being added here)